terminus-op/templates/tdb-persistentVolume.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: {{ .Release.Name}}-persistentvolume
+spec:
+  claimRef:
+    name: {{ .Release.Name}}-pvc
+  storageClassName: terminus-op-storage-class
+  capacity:
+    storage: {{ .Values.persistent.VolumeStorageSize}}
+  volumeMode: Filesystem
+  accessModes:
+    - ReadWriteOnce
+  persistentVolumeReclaimPolicy: Recycle
+  storageClassName: terminus-op-storage-class
+  mountOptions:
+    - ssd
+    - nfsvers=4.1
+  nfs:

```

Comment:
Shouldn't this be dynamic? Perhaps people want other storage mechanisms than nfs. And we should probably not hardcode the path and server. Also, we probably don't want the path to be hardcoded to /tmp either.

Reply:
This is the part I needed some help with, I hardcoded this for a local PC, I want to change this but I am not sure how to go about it.